clients/authentication/super-agent-scopes/index.ts

```diff
@@ -25,7 +25,9 @@ class SuperAgentsScopes {
       .filter((s: string) => isAgentScope(s)) as IAgentScope[];
   };
 
-  mapResponseToAgentScopes = (response: any) =>
+  mapResponseToAgentScopes = (
+    response: IAgentRecord[]
+  ): { [key: string]: IAgentScope[] } =>
     response
       .filter((r: any) => r.actif === true)
```

Comment:
```suggestion
      .filter((r) => r.actif === true)
```


Type inference will do the job